solidity/contracts/transformers/wstETHTransformer.sol

```diff
@@ -23,10 +23,16 @@ contract wstETHTransformer is BaseTransformer {
   }
 
   /// @inheritdoc ITransformer
-  function calculateTransformToUnderlying(address, uint256 _amountDependent) external view returns (UnderlyingAmount[] memory) {}
+  function calculateTransformToUnderlying(address, uint256 _amountDependent) external view returns (UnderlyingAmount[] memory) {
+    uint256 _amountUnderlying = stETH.getPooledEthByShares(_amountDependent);
+    return _toSingletonArray(stETH, _amountUnderlying);
+  }
 
   /// @inheritdoc ITransformer
-  function calculateTransformToDependent(address, UnderlyingAmount[] calldata _underlying) external view returns (uint256 _amountDependent) {}
+  function calculateTransformToDependent(address, UnderlyingAmount[] calldata _underlying) external view returns (uint256 _amountDependent) {
+    if (_underlying.length != 1) revert InvalidUnderlyingInput();
+    _amountDependent = stETH.getSharesByPooledEth(_underlying[0].amount);
+  }
```

Comment:
Same question, the input on underlying amount wouldn't it be denominated in `stETH`? While `getSharesByPooledEth` [docs specify](https://github.com/lidofinance/docs/blob/3231ce731677e53d7fd0a1a57bebbdbe6ba4538b/docs/contracts/lido.md#getsharesbypooledeth) that the input for this is `ethAmount`.